contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/__tests__/plugins/formulas/markdownFormulaElement.spec.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @jest-environment jest-environment-jsdom-sixteen
```

Comment:
Jsdom@16 is the first version of jsdom to support custom elements, but Jest uses an older version of jsdom (version 11 I think).  I tried upgrading to a new version of jsdom for all tests, but some of the other tests broke.